pkg/okteto/preview.go

```diff
@@ -227,9 +227,6 @@ func (c *OktetoClient) DestroyPreview(ctx context.Context, name string) error {
 	}
 
 	err := mutate(ctx, &mutation, variables, c.client)
-	if oktetoErrors.IsNotFound(err) {
```

Comment:
@AgustinRamiroDiaz why we need this change? Destroy is usually an idempotent operation

Reply:
It's to give an explicit error to the user. I forgot to say that it it related to [this](https://github.com/okteto/okteto/issues/2343) issue

Reply:
@AgustinRamiroDiaz but is it an error? I would show an info message here, but I wouldn't fail

Reply:
I get what you are saying. What benefits do we get from not failing in the Cobra command?

Reply:
When we run this command from a Github Action we shouldn't fail if the preview doesn't exists. It can give the user an error that the user shouldn't be aware of

Reply:
@jmacelroy that's a good point, on the cli side we don't know if `not-found` if you don't have access to the preview, or if the preview doesn't exist...
Yes, I think we have to return an error here

Reply:
We can log with oktetoLog.Information. In this way users know they had a problem and the cli doesn't fail, letting other workflows still functional

Reply:
@AgustinRamiroDiaz I would go with the error as it's currently in the PR